pkgs/by-name/wa/waywall/package.nix

```
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  egl-wayland,
+  libglvnd,
+  libspng,
+  libxkbcommon,
+  luajit,
+  mesa_glu,
+  meson,
+  ninja,
+  pkg-config,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  xorg,
+  xwayland,
+}:
+stdenv.mkDerivation {
+  pname = "waywall";
+  version = "0-unstable-2025-02-07";
+
+  src = fetchFromGitHub {
+    owner = "tesselslate";
+    repo = "waywall";
+    rev = "be96e20";
+    hash = "sha256-77GbBzHjyZuauhl0vlguUS/7jBT4qNjOLYGWBVTPjEY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    meson
+    ninja
+    pkg-config
+    wayland-scanner
+  ];
+
+  buildInputs = [
+    egl-wayland
+    libglvnd
+    libspng
+    libxkbcommon
+    luajit
+    mesa_glu
+    meson
+    ninja
+    pkg-config
+    wayland
+    wayland-protocols
+    xorg.libxcb
+    xwayland
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp waywall/waywall $out/bin
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
See https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242